helm/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 image:
   repository: public.ecr.aws/aws-controllers-k8s/elasticache-controller
-  tag: 0.1.0
+  tag: 0.0.30
```

Comment:
Can you please feetch the tags from upstream and regenerate the controller?

Reply:
I'm not done making the full changes yet. I was just testing the pipeline.